pom.xml

```diff
@@ -286,7 +286,7 @@
         <version.javax.json.bind.api>1.0</version.javax.json.bind.api>
         <version.javax.jws.jsr181-api>1.0-MR1</version.javax.jws.jsr181-api>
         <version.javax.mail>1.6.2</version.javax.mail>
-        <version.javax.persistence>2.2</version.javax.persistence>
+        <version.jakarta.persistence>2.2.3</version.jakarta.persistence>
```

Comment:
Please put this in the new alphabetical position.

Reply:
Thanks, moved to just before jakarta.security.enterprise